src/settings/NoticePolicy/components/EditSections/components/NoticeCard/NoticeCard.test.js

```diff
@@ -0,0 +1,396 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  fireEvent,
+} from '@testing-library/react';
+
+import '../../../../../../../test/jest/__mock__';
+
+import { Field } from 'react-final-form';
+
+import {
+  IconButton,
+  MessageBanner,
+  Select,
+  RadioButton,
+} from '@folio/stripes/components';
+
+import NoticeCard from './NoticeCard';
+import Period from '../../../../../components/Period';
+
+import {
+  noticesFormats,
+  noticesFrequency,
+  noticesIntervalPeriods,
+} from '../../../../../../constants';
+
+import { componentPropsCheck } from '../../../../../../../test/jest/helpers';
+
+jest.mock('../../../../../utils/options-generator', () => jest.fn((func, data) => data));
+jest.mock('../../../../utils/notice-description', () => jest.fn(data => data));
+jest.mock('../../../../../components/Period', () => jest.fn(({ 'data-testid': testId }) => (
+  <div data-testid={testId} />
+)));
+
+IconButton.mockImplementation(({ icon, onClick }) => (
+  // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+  <div
+    onClick={onClick}
+  >
+    {`iconButton-${icon}`}
+  </div>
+));
+Field.mockImplementation(({ 'data-testid': testId, label }) => (
+  <div data-testid={testId}>
+    {label}
+  </div>
+));
```

Comment:
You can remove it. It is already implemented [here](https://github.com/folio-org/ui-circulation/blob/master/test/jest/__mock__/finalFormComponents.mock.js)